specs/006-requirement-atomization/contracts/atomize-api.md

```
@@ -0,0 +1,399 @@
+
+**Feature**: 006-requirement-atomization
+**Module**: `src/parse/atomize.rs`
+**Generated**: 2026-02-11
+**Source**: [spec.md](../spec.md), [AR-006](../../../docs/AR/006-ar-requirement-atomization.md)
+
+## Public API
+
+### `atomize_document`
+
+**Signature**:
+```rust
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)